### ISSUE TITLE
Why is `JET` a direct dependency?

### ISSUE THREAD

**Issue by avik-pal** [compintell/Tapir.jl#117]
I noticed JET is a direct dependency for the package. From the use in code it seems like it is just for testing, so that can be refactored out?

```julia
(wip) pkg> st
Status `/mnt/research/lux/Lux.jl/wip/Project.toml`
  [b0b7db55] ComponentArrays v0.15.11
  [b2108857] Lux v0.5.34 `..`
  [07d77754] Tapir v0.1.2
  [e88e6eb3] Zygote v0.6.69

(wip) pkg> why JET
  Tapir → JET
```

**Comment by willtebbutt** [comment]
You're correct re it only being used for testing. At some point before we properly release we'll factor it out.

**Comment by willtebbutt** [comment]
Resolved by #200
